benchmark-framework/src/main/java/io/openmessaging/benchmark/worker/WorkerHandler.java

```diff
@@ -164,6 +166,15 @@ static ByteBuffer serializeHistogram(Histogram histo, ByteBuffer buffer) {
     private void handlePeriodStats(Context ctx) throws Exception {
         PeriodStats stats = localWorker.getPeriodStats();
 
+        // We output the stats for the current worker to the log, in the same way as the driver, in order
+        // to have a per-worker record of these values (since the driver currently always aggregates metrics
+        // from all workers).
+        long now = System.nanoTime();
+        if (lastPeriodStatsTime != -1) {
+            WorkloadGenerator.printPeriodStats(stats, (now - lastPeriodStatsTime) / 1.e9, 0);
```

Comment:
there zero here hardcodes backlog to zero: we can't calculate it from the worker